docs/sources/dashboards/export-import.md

```diff
@@ -25,6 +25,11 @@ add template variables for things like a metric prefix (use constant variable) a
 A template variable of the type `Constant` will automatically be hidden in
 the dashboard, and will also be added as a required input when the dashboard is imported.
 
+### Export with default values removed
+
+If you want to export a dashboard without default values in order to reduce the exported json file size, it is a good idea to check the toggle "Export with default values removed".
+
+When import, the removed default values would be added automatically to the dashboard.
```

Comment:
Some language fixes, JSON should be written in all-caps as it's an acronym:
```suggestion
If you want to export a dashboard without default values in order to reduce the exported JSON file size, it is a good idea to check the toggle "Export with default values removed".

When importing, the removed default values will be added automatically to the dashboard.
```

Reply:
Agree, JSON should be all caps. Documentation best practice says you should avoid future tense if you can. How about: 

"During import, the removed default values automatically added back to the dashboard."

Reply:
There's a typo in @achatterjee-grafana's suggestion, it should probably be "During import, the removed default values are automatically added back to the dashboard.".